test/integration/strainer_test.rb

```diff
@@ -0,0 +1,15 @@
+require 'test_helper'
+
+class StrainerTest < Minitest::Test
+  include Liquid
+
+  def test_add_filter_when_wrong_filter_class
```

Comment:
This test looks like a unit test to me, not an integration test... Why is it in `test/integration/`?